### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by yosaddis**
# In this feature I have done the following.

Initialize a new Ruby on Rails project with a connection to PostgreSQL. 

- Add Users table. 
- Add Posts table. 
- Add Comments table. 
- Add Likes table. 
- Generate schema.rb.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  end
+
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "photo"
+    t.text "bio"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.integer "posts_counter", default: 0
+  end
```
Comment (algerina) [inline db/schema.rb:54]: -Well done adding the tables and the foreign keys following the ERD diagram however it is required to 
- [ ] Add User, Post, Comment and Like models inside `app/models/ `directory.
- [ ] Set the associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).The associations between models are set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- [ ] Add the required methods in every model.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,172 @@
...
+
+I would like to thank Microverse community.
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (algerina) [inline README.md:170]: - [ ] The link of the MIT license is broken, kindly fix it.

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @yosaddis ,

Great job so far!  However you  have some issues to work on.



## CHANGES REQUIRED:muscle::clap:

Check the comments under the review.



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

File: README.md (hunk 2 above)
Comment (yosaddis) [reply README.md:170]: @algerina  Thank you for your review. I submitted the link in the wrong day. I have submitted it again to get approval and continue with your comments. Thank you.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @yosaddis 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
✔ Created migration files in order to set up database tables.
✔ Database adapter is using Postgresql


---
## Important 🙏
I wanted to provide some feedback on your recent submission. It looks like there might have been a bit of confusion, possibly due to some overlooked details.

Before you submit your work next time, I'd like to encourage you to take a few extra moments to review everything. This can help ensure that everything is accurate and complete, which in turn will help us avoid any misunderstandings.

Remember, we're all here to learn and improve together, and paying attention to these small details can make a big difference in the quality of our work. If you have any questions or need any assistance, feel free to reach out. I'm here to help! 🙏



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (algerina) [reply README.md:170]: You're welcome.

**Pull request merged**